O/OpenBLAS/common.jl

```diff
@@ -42,9 +43,14 @@ function openblas_script(;num_64bit_threads=32, kwargs...)
     flags+=(NO_STATIC=1)
 
     if [[ ${nbits} == 64 ]] && [[ ${target} != aarch64* ]]; then
-        # If we're building for a 64-bit platform (that is not aarch64), engage ILP64
-        LIBPREFIX=libopenblas64_
-        flags+=(INTERFACE64=1 SYMBOLSUFFIX=64_)
+        if [[ ${openblas32} == 1 ]]; then
```

Comment:
Shouldn't this be
```suggestion
        if [[ ${OPENBLAS32} == 1 ]]; then
```
?  That's the name of the variable defined in the script at line 29.